CONTRIBUTING.md

```diff
@@ -60,37 +64,39 @@ into your branch:
     git checkout new_branch_name
     git merge main
     ```
-    You may need to fix merge conflicts for both of these merge
-    commands.
-1. Finally, push your change to your clone
+    You may need to fix [merge
+    conflicts](https://docs.github.com/articles/about-merge-conflicts)
+    during one or both of these merge processes.
+1. Finally, push your changes to your forked copy of the Cirq repo on GitHub:
     ```shell
     git push origin new_branch_name
     ```
-1. Now when you navigate to the cirq page on github,
-[https://github.com/quantumlib/cirq](https://github.com/quantumlib/cirq)
+1. Now when you navigate to the Cirq repository on GitHub
+([https://github.com/quantumlib/cirq](https://github.com/quantumlib/cirq)),
 you should see the option to create a new pull request from
-your clone repository.  Alternatively you can create the pull request
+your clone repository.  Alternatively, you can create the pull request
 by navigating to the "Pull requests" tab in the page, and selecting
 the appropriate branches.
-1. The reviewer will comment on your code and may ask for changes,
-you can perform these locally, and then push the new commit following
-the same process as above.
+1. A reviewer will comment on your code and may ask for changes.
+You can perform the necessary changes locally, and then push the new commit
+following the same process as above.
 
 ## Development Environment Setup
 
-Please refer to our [development page](docs/dev/development.md) for instructions
-on setting up your local development environment.
+Please refer to our [development page](docs/dev/development.md) for
+instructions on setting up your local development environment.
 
 ## Code Testing Standards
 
 When a pull request is created or updated, various automatic checks will
-run to ensure that the change won't break Cirq and meets our coding standards.
+run on GitHub to ensure that the changes won't break Cirq, as well as to make
+sure they meet the Cirq project's coding standards.
 
-Cirq contains a continuous integration tool to verify testing.  See our
+Cirq includes a continuous integration tool to perform testing.  See our
 [development page](docs/dev/development.md) on how to run the continuous
 integration checks locally.
 
-Please be aware of the following code standards that will be applied to any
+Please be aware of the following coding standards that will be applied to any
```

Comment:
Can you please remove the `/en/latest` suffix from the pytest and pytest-cov URLs below?
Their top pages already redirect to `.../en/latest` and are less likely to become invalid.

Reply:
These happen when I copy-paste the links. You’re right that they should be removed. This is something we should put into linter rules.